app/views/student_flashcards/show.html.erb

```diff
@@ -0,0 +1,16 @@
+show page for student flashcard
+
+<%= link_to 'Back', student_student_flashcard_sets_student_flashcards_path, class: "btn btn-outline-dark m-2" %>
+<div class="container mt-3">
```

Comment:
I wouldn't put the mt-3 in the container out of standardization (in all other pages with Back button is not there, and actually, I removed it cos you thought it was too much!)